tests/test_spheropolygon.py

```diff
@@ -110,6 +111,21 @@ def test_area(unit_rounded_square):
     assert shape.area == area
 
 
+@given(floats(0.1, 1000))
```

Comment:
To avoid any ambiguity, you can make this explicit with a keyword argument.
```suggestion
@given(area=floats(0.1, 1000))
```